apps/charterafrica/src/payload/blocks/FeaturedPost.js

```diff
@@ -0,0 +1,54 @@
+const FeaturedPost = {
+  slug: "featured-post",
+  fields: [
```

Comment:
Don't think this is the right approach. Content editors should just mark a post as featured or select a post as featured. They shouldn't need to retype all of this.